types/didyoumean/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for didyoumean 1.2.1
```

Comment:
FYI, this is what’s causing the builds to fail—the version number should only be major/minor, no patch number.